atlp-rwanda/octopus-bn-frontend
# 172674154 - Present a user with a password reset form

#### What does this PR do?
- It creates a form for a user to reset their password. 

#### Description of Task to be completed?
- create a **New Password**  input field
- create a **Confirm Password** input field.

#### How should this be manually tested?
- clone the repo, then checkout to this branch ``ft-finish-password-reset-172674154``
- run `npm install` to install all project dependencies
- run `npm run dev` to start the development server
- sign up with your ``personal email`` to see how it works
- got to ``localhost:your_port/forgot-password`` 
- enter your email address.
- follow the instructions
- go to ``localhost:your_port/reset-password``  to reset your password.

#### Any screenshots? 
<img width="1440" alt="Screen Shot 2020-05-12 at 11 11 23" src="https://user-images.githubusercontent.com/25052255/81665735-5b135480-9441-11ea-985c-1efeac0f4f74.png">

#### Related Pivotal Tracker story(ies)
[ #172674154](https://www.pivotaltracker.com/story/show/172674154)
